utilities/flask_responses.py

```diff
@@ -1,25 +1,27 @@
 from http import HTTPStatus
-from utilities.types import FlaskResponseType, JSONType
+from flask.typing import ResponseReturnValue
+from utilities.types import JSONType
+import json
 
 
 class FlaskResponses():
 
     @classmethod
-    def not_implemented_yet(cls) -> FlaskResponseType:
+    def not_implemented_yet(cls) -> ResponseReturnValue:
         return {'error': "not implemented yet"}, HTTPStatus.NOT_IMPLEMENTED
 
     @classmethod
-    def success(cls, data: JSONType) -> FlaskResponseType:
-        return data, HTTPStatus.OK
+    def success(cls, data: JSONType) -> ResponseReturnValue:
+        return json.dumps(data), HTTPStatus.OK
```

Comment:
This does change output when using `curl`. I'm not sure if this will cause issues when we implement the frontend.

Steps to reproduce:

1. Generate id token. See https://github.com/castlepointanime/backend/issues/31 for details
3. Set the ID token to a variable `token` in shell
3. Run the backend and run this command

```
curl -X GET -H "Authorization: Bearer $token" -H 'Content-Type: application/json' localhost:3001/me
```

Output before MR:
```
{
    "name": "test",
    "email": "test@mail.com",
    "database": null
}
```

Output after MR:
```
"{\"name\": \"test\", \"email\": \"test@mail.com\", \"database\": null}"
```

Reply:
nevermind. I did a test and this isn't a problem for browsers.

Reply:
did this change the headers in the response? i'm guessing if the headers don't say its a json curl might default to displaying as a string

Reply:
No it doesn't change the headers. My guess is that `json.dumps()` changes the formatting to include `\`

After Changes:
```
Internal server errorgcarvellas@razer-blade-15:~$ curl -v -X GET localhost:3001/health
Note: Unnecessary use of -X or --request, GET is already inferred.
* processing: localhost:3001/health
*   Trying [::1]:3001...
* Connected to localhost (::1) port 3001
> GET /health HTTP/1.1
> Host: localhost:3001
> User-Agent: curl/8.2.1
> Accept: */*
> 
< HTTP/1.1 200 OK
< Server: gunicorn
< Date: Thu, 14 Sep 2023 18:57:58 GMT
< Connection: keep-alive
< Content-Type: application/json
< Content-Length: 9
< Access-Control-Allow-Origin: *
< 
"\"ok\""
```

Before Changes:
```
gcarvellas@razer-blade-15:~$ curl -v -X GET localhost:3001/health
Note: Unnecessary use of -X or --request, GET is already inferred.
* processing: localhost:3001/health
*   Trying [::1]:3001...
* Connected to localhost (::1) port 3001
> GET /health HTTP/1.1
> Host: localhost:3001
> User-Agent: curl/8.2.1
> Accept: */*
> 
< HTTP/1.1 200 OK
< Server: gunicorn
< Date: Thu, 14 Sep 2023 18:58:40 GMT
< Connection: keep-alive
< Content-Type: application/json
< Content-Length: 5
< Access-Control-Allow-Origin: *
< 
"ok"
```